README.md

```diff
@@ -3,25 +3,40 @@ Azure Functions prepared for the Mapping Viz Tools project. This repo contains [
 
-### yarn install
+### Local database setup
+In order to run the app on your local machine, install PostgreSQL 9.5 and run the PostgreSQL service locally. Then, create a database called `mapping_viz_local`. Once that database is created, execute the db/schema.sql file and db/test_data.sql file on the `mapping_viz_local` database. You can do so by connecting to the `mapping_viz_local` database via `psql` on the command line, then running the schema.sql file using the `\include` command, e.g. `\include path\to\file\schema.sql` (see more on how to use the `\include` command in the [psql documentation](https://www.postgresql.org/docs/9.2/app-psql.html)). Next, populate the database with test data by running `\include path\to\file\test_data.sql`.
+
+### `yarn install`
 To install the necessary dependencies
 
-### yarn test
+### `yarn test`
 To test the app
 
-### yarn start
+### `yarn start`
 To start the server. You an then try hitting the available endpoints (see terminal).
 
 ### Connection Strings
 For security, connection strings are set on the Azure Portal. Check out the `PG_DB_CONNECTION_STRING` ENV variable for this function when logged into the portal. For local testing, this variable can be set in the `local.settings.json` file. Just be careful not to commit any sensitive information when testing. 
 
-### Deployment
+## Deployment
 Deploying an Azure Function is simple, using the Azure VS Code extension. You can follow steps [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-first-function-vs-code?pivots=programming-language-javascript#publish-the-project-to-azure) on how to publish a function. 
 
 #### Staging or Production?
-Although you could use Azure [Slots](https://docs.microsoft.com/en-us/azure/azure-functions/functions-deployment-slots) to handle different environments, you can also deploy the same function at a different location (eg. `mapping-viz-functions-staging` vs `mapping-viz-functions`) and set a different connection string at each endpoint to handle staging and production databases.
\ No newline at end of file
+Although you could use [Azure Slots](https://docs.microsoft.com/en-us/azure/azure-functions/functions-deployment-slots) to handle different environments, you can also deploy the same function at a different location (eg. `mapping-viz-functions-staging` vs `mapping-viz-functions`) and set a different connection string at each endpoint to handle staging and production databases.
+
+## Database Maintenance
+The current schema is saved in db/schema_staging.sql and db/schema_production (no difference besides owner). This files are generated using [pg_dump](https://www.postgresql.org/docs/9.1/app-pgdump.html) (version 9.5.23). There are also files containing test data for local development, and the full dataset for back-up purposes - those live in db/data. The full_data.sql file is compressed using the `pg_dump` command's built in compression (using the flag `--compress=9`).
```

Comment:
link is pointing to the 9.1 doc versions, https://www.postgresql.org/docs/9.1/app-pgdump.html should be 9.5

Reply:
Good catch. Updated.